src/enum_net.cpp

```diff
@@ -980,6 +980,7 @@ namespace libtorrent
 		GetIfEntry_t GetIfEntry = (GetIfEntry_t)GetProcAddress(iphlp, "GetIfEntry");
 		if (!GetIfEntry)
 		{
+			FreeLibrary(iphlp);
```

Comment:
hm.. I was under the impression that the library was only loaded once, into a static variable. I guess that's not the case, but that also means this library is loaded for every call here, which doesn't seem very good